Seminar14/test_users.py

```diff
@@ -0,0 +1,40 @@
+# Задание №6
+# На семинаре 13 был создан проект по работе с
+# пользователями (имя, id, уровень).
+# Напишите 3-7 тестов pytest для данного проекта.
+# Используйте фикстуры.
+
+import pytest
+from Task4 import User1
+from Task5 import Project
+
+
+@pytest.fixture
+def user_create():
+    return User1(123, 'Kate')
+    
+    
+@pytest.fixture
+def project():
+    return Project.load('users.json')
+
+
+def test_new_User():
+    assert User1(123, 'Kate').__str__() == "self.id = 123, self.name = 'Kate', self.level=None"
+
+
+def test_eq(user_create):
+    assert (user_create == User1(123, 'Kate')) == True
```

Comment:
```suggestion
    assert user_create == User1(123, 'Kate')
```
сообщение бы еще добавить